client/src/containers/CategoryFilterContainer.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import ListGroup from 'react-bootstrap/ListGroup';
+// import Container from 'react-bootstrap/Container';
+
+
+const CategoryFilterContainer = () => {
+    const [activeButton, setActiveButton] = useState(null);
+
+    const handleButtonClick = (buttonType) => {
+        if (buttonType === activeButton) {
+          setActiveButton(null);
+        } else {
+          setActiveButton(buttonType);
+        }
+    };
+
+    return (
+    <div>
+    <Form>
+        <Form.Group>
+            <Form.Control as="input" type="text" placeholder="Filter by:" />
+        </Form.Group>
+            <Button variant="primary" type="button" onClick={() => handleBurronClick('Title')}>Title</Button>
+            <Button variant="primary" type="button" onClick={() => handleButtonClick('Company')}>Company</Button>
+            <Button variant="primary" type="button" onClick={() => handleButtonClick('Salary Range')}>Salary Range</Button>
+            <Button variant="primary" type="button" onClick={() => handleButtonClick('Level')}>Level</Button>
+            <Button variant="primary" type="button" onClick={() => handleButtonClick('Status')}>Status</Button>
+    </Form>
+    {activeButton && (
+        <Card style={{ width: '18rem' }}>
+          <ListGroup variant="flush">
+            {activeButton === 'Level' && ( 
+            <div>
```

Comment:
I believe you would have to nest each of these groups of 'ListGroup.Item's in their own 'ListGroup' component instead of one big 'ListGroup' separated by divs. Might be wrong